### DISCUSSION TITLE
Café

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#341] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/tirinhas/cafe/

**Comment by brunoaugusto** [comment]
Uns dois anos atrás eu topava fácil

**Comment by one-eyed-willy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [segunda-feira, 20 de junho de 2022](<https://www.vish.blog.br/imagens/tirinhas/cafe/#comment_1160610407>):

> Uns dois anos atrás eu topava fácil

Mas café é vida, cara
